datastore/tasks/tasks.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START all]
+
+// A simple command-line task list manager to demonstrate using the
+// google.golang.org/cloud/datastore package.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/cloud/datastore"
+)
+
+func main() {
+	projID := os.Getenv("DATASTORE_PROJECT_ID")
```

Comment:
For v1beta3, DATASTORE_PROJECT_ID needs to be used since it is consistent with what the emulator expects and what the other v1beta3 client lib samples / docs are using.

For post-v1beta3 releases, it can be revisited, but whatever env var is used, it should use _PROJECT_ instead of _DATASET_ (moving away from "dataset").